src/aitsi/m3spin/pkb/interfaces/Parent.java

```diff
@@ -8,6 +8,8 @@ public interface Parent {
 
     List<TNode> getParentedBy(TNode p);
 
+    TNode getParent(TNode c);
```

Comment:
Tak na przyszłość powiem, że nie powinno się używać jednoliterkowych nazw zmiennych w interfejsach bo... Jedna literka jest mało opisowa, a z interfejsów korzystają przede wszystkim programiści, którzy ich nie pisali 😅 Właściwie, to w ogóle się nie powinno tak nazywać zmiennych, chyba że jest to oczywiste co ta jedna litera oznacza, albo inaczej się nie da

Reply:
+1 (zasłużyło na dodatkowy komentarz XD)

Reply:
To tutaj do Jarząbka proszę emaila pisać, interfejsów nikt z palca nie wpisywał xD

Reply:
Fixed

Reply:
Niby taki profesor, a takie rzeczy robi xD chyba że skrócił nazwy, żeby mu się na slajdzie zmieściły 🤔